src/EditorFeatures/Core/Implementation/Classification/EditorSemanticClassificationCacheService.cs

```diff
@@ -7,41 +7,29 @@
 using System.Composition;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Classification;
-using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.PersistentStorage;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.SemanticClassificationCache
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
 {
     [ExportWorkspaceService(typeof(ISemanticClassificationCacheService), ServiceLayer.Host), Shared]
-    internal class VisualStudioSemanticClassificationCacheService
-        : ForegroundThreadAffinitizedObject, ISemanticClassificationCacheService
+    internal class EditorSemanticClassificationCacheService : ISemanticClassificationCacheService
```

Comment:
>EditorSemanticClassificationCacheService [](http://example.com/codeflow?start=19&length=40)

Does this have any dependency on the Editor/UI? I don't see any. Should this be in Features?

Reply:
that works for me.

Reply:
Moved down.